spec/integration/settings_component_spec.rb

```diff
@@ -73,16 +73,16 @@
       ENV.delete('COERCIBLE_VALUE')
     end
 
-    it 'raises InvalidSettingValueError with meaningful message' do
+    it 'raises InvalidSettingsError with meaningful message' do
       expect {
         settings.integer_value
       }.to raise_error(
-        Dry::System::InvalidSettingValueError,
+        Dry::System::InvalidSettingsError,
         <<~EOF
-          Invalid setting values for:
+          Could not initialize settings. The following settings were invalid:
 
-          integer_value type?(Integer, "foo")
-          coercible_value invalid value for Integer(): "foo"
+          integer_value: type?(Integer, "foo")
+          coercible_value: invalid value for Integer(): "foo"
```

Comment:
Next job for us (definitely outside of the scope of this PR) is to make dry-types error messages less cryptic than `type?(Integer, "foo")` 😅

Reply:
Couldn't agree more with you 😄